commit 3f293c7caaefc2c37b61e44e8ebd5a7f1c554afb
Author: Dario Faggioli <dario.faggioli@citrix.com>
Date:   Thu Feb 25 13:03:04 2016 +0100

    public: typo: use ' as apostrophe in grant_table.h
    
    If grep 2.23 is installed, build fails like this:
    ...
    mkdir -p compat
    grep -v 'DEFINE_XEN_GUEST_HANDLE(long)' public/grant_table.h | \
    python /home/SOURCES/xen/xen/xen.git/xen/tools/compat-build-source.py >compat/grant_table.c.new
    mv -f compat/grant_table.c.new compat/grant_table.c
    gcc  ... -o compat/grant_table.i compat/grant_table.c
    compat/grant_table.c:33:1: error: unterminated comment
     /*
     ^
    compat/grant_table.c:28:0: error: unterminated #ifndef
     #ifndef __XEN_PUBLIC_GRANT_TABLE_H__
     ^
    Makefile:62: recipe for target 'compat/grant_table.i' failed
    make[3]: *** [compat/grant_table.i] Error 1
    rm compat/grant_table.c
    make[3]: Leaving directory '/home/SOURCES/xen/xen/xen.git/xen/include'
    ...
    
    This is because grant_table.h contains this (note the
    apostrophe): "granters memory", and `grep -v', in version
    2.23, stops processing the file (while, for instance,
    until 2.22, this was not happening).
    
    Although the above behavior is likely an issue in grep,
    (https://debbugs.gnu.org/cgi/bugreport.cgi?bug=22461)
    I think we better switch to using " ' " in that line
    anyway, as we do basically everywhere else (even in
    the same file).
    
    Signed-off-by: Dario Faggioli <dario.faggioli@citrix.com>

diff --git a/xen/include/public/grant_table.h b/xen/include/public/grant_table.h
index e9393fd..e5f04ec 100644
--- a/xen/include/public/grant_table.h
+++ b/xen/include/public/grant_table.h
@@ -43,7 +43,7 @@
  * table are identified by grant references. A grant reference is an
  * integer, which indexes into the grant table. It acts as a
  * capability which the grantee can use to perform operations on the
- * granter’s memory.
+ * granter's memory.
  *
  * This capability-based system allows shared-memory communications
  * between unprivileged domains. A grant reference also encapsulates
